.travis.yml

```diff
@@ -4,7 +4,7 @@ node_js:
   - "0.12"
   - "0.10"
 script:
-  - npm test
```

Comment:
@hx-markterry suggested in my pr these should go in a `npm run ci` script in package.json